application/src/test/java/uk/nhs/hee/tis/revalidation/service/RecommendationServiceTest.java

```diff
@@ -587,6 +588,22 @@ void shouldNotUpdateRecommendationWhenSubmitFail() {
     verify(recommendationRepository, times(0)).save(recommendation);
   }
 
+  @Test
+  void shouldNotUpdateRecommendationWhenSubmitAttemptOnEarlyDeferral() {
+    final var recommendation = buildRecommendation(gmcNumber1, recommendationId, status,
+        UNDER_REVIEW);
+    final var userProfileDto = buildRoUserProfileDto(gmcNumber1);
+    when(doctorsForDBRepository.findById(gmcNumber1)).thenReturn(Optional.of(doctorsForDB));
+    when(recommendationRepository.findByIdAndGmcNumber(recommendationId, gmcNumber1))
+        .thenReturn(recommendation);
+    when(gmcClientService.submitToGmc(doctorsForDB, recommendation, userProfileDto))
+        .thenReturn(buildRecommendationV2Response(DOCTOR_DEFERRAL_ON_EARLY_DATE.getCode()));
+
+    assertThrows(RecommendationException.class, () -> recommendationService
+        .submitRecommendation(recommendationId, gmcNumber1, userProfileDto));
```

Comment:
There's nothing here to actually assert the new code.
As well as here, you could consider how the Controller responds `RecommendationControllerTest.java`